layouts/shortcodes/home/past_events.html

```diff
@@ -0,0 +1,64 @@
+
+{{$currLang:=.Page.Lang }}
+{{$langBase:=index .Site.Data $currLang}}
+{{$events := (index $langBase).conferences.items}}
+
+<section class="past-events row">
+  <div class="container">
+  
+  <style>
```

Comment:
Can we put this in solstice-assets or in custom.less?

Reply:
You seem to be looking at an old commit, this was removed a few commits ago